static/css/style.css

```diff
@@ -957,6 +957,17 @@ input:checked + .menu-switch-slider:before {
     float: right;
 }
 
+#file-upload-link {
```

Comment:
Let's create a generic class for this as we have more link in other modals that could use this.

Reply:
Do you mean the links created by https://github.com/bbcmicrobit/PythonEditor/blob/3b27fe802ba8c132a90ca59858c309a06cbee547/python-main.js#L1740 and 
https://github.com/bbcmicrobit/PythonEditor/blob/3b27fe802ba8c132a90ca59858c309a06cbee547/python-main.js#L1445

If so, shall I update the JS to use buttons and as they are using a different class https://github.com/bbcmicrobit/PythonEditor/blob/3b27fe802ba8c132a90ca59858c309a06cbee547/static/css/style.css#L807 update this to use a single class for all error modal links?

Reply:
Yes, the WebUSB error modal and the modalMsg modal.

> https://github.com/bbcmicrobit/PythonEditor/blob/3b27fe802ba8c132a90ca59858c309a06cbee547/static/css/style.css#L807 update this to use a single class for all error modal links?

Not necessarily, that CSS class set-ups the padding and things like that, which would apply as before. 

The class class to replace #file-upload-link should only contain the CSS required to make a button look exactly like a link, we then add that css class to all the links reworked into buttons, and keep all the old classes that were already attached to them.

Reply:
I've updated this, but I'm not stuck at how to now trigger the click for the webUSB not availble modal in' Find Out More'